application/src/main/java/org/thingsboard/server/service/entitiy/ota/DefaultTbOtaPackageService.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright © 2016-2022 The Thingsboard Authors
- *
+ * <p>
```

Comment:
This is wrong. The build will fail. Execute "mvn license:format" to fix

Reply:
my bad, will fix